app/templates/main/browse-series.html

```diff
@@ -1,4 +1,25 @@
 {% if view_type == "desktop" %}
+    <!-- BREAD CRUMB -->
```

Comment:
we could make a breadcrumbs html file which takes in variable:

something like 
https://github.com/nationalarchives/ds-caselaw-editor-ui/blob/main/ds_caselaw_editor_ui/templates/includes/document_breadcrumbs.html

but I think we could make it simpler by making it a for loop with the last element treated differently

e.g. for series
parent_structure_list = [{"Everything": everyting_url, {"Body A": body_a_url}]
end_node = "Series Name"

with this structure we could easily make the breadcrumbs a generic include.

Could even have it at the browse.html level and then just pass in `parent_structure_list`, `end_node` as parameters to the render function

Reply:
separated breadcrumb to a new template and updated all browse templates (incl. record template) to use that template